gallery/src/components/NxTransferListHalf/NxTransferListHalfDisableTransferExample.tsx

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React, { useState } from 'react';
+import { DataItem, NxFontAwesomeIcon, NxTransferListHalf } from '@sonatype/react-shared-components';
+import { map, range, prepend } from 'ramda';
+import { faArrowsAltH } from '@fortawesome/free-solid-svg-icons';
+
+const Items: DataItem<number>[] = prepend(
+    {
+      id: 0,
+      displayName: <><NxFontAwesomeIcon icon={faArrowsAltH} /><span>Loooooooooooooooooooooooooong Name</span></>
+    },
+    map<number, DataItem<number>>(i => ({ id: i, displayName: `Item ${i}` }), range(1, 101))
+);
+
+export default function NxTransferListHalfDisableTransferExample() {
+  const [filter, setFilter] = useState('');
+
+  return <NxTransferListHalf label="Example Items"
+                             filterValue={filter}
+                             onFilterChange={setFilter}
+                             showMoveAll={false}
+                             onMoveAll={() => {}}
```

Comment:
It seems a little goofy that `showMoveAll` and `onMoveAll` need to be provided in this case.  For that matter, it seems a little goofy that `onMoveAll` needs to be provided any time `showMoveAll` is false.  IMO those two props should also be made optional and excluded from this example.

Reply:
Updated here 8a6881fef17ec3292383fa277f21d76019b90b93.
Further on this, since there won't be selected/unselected items in this case, should `isSelected` prop be optional as well?

Reply:
> Further on this, since there won't be selected/unselected items in this case, should isSelected prop be optional as well?

Sure

Reply:
Updated here 8b6823bcb2af5335fecb102363cf56f47854aed8